indra/llcharacter/tests/llik_test.cpp

```diff
@@ -1494,19 +1517,20 @@ namespace tut
         // solve
         //solver.enableDebugIfPossible();
         solver.updateJointConfigs(configs);
-        F32 max_error = solver.solve();
+        F32 error = solver.solve();
 
         // check results
         // Note; this test does not quite reach ACCEPTABLE_ERROR after 16 iterations
         // however it gets close
-        F32 allowable_error = 0.033f;
-        for (size_t i = 0; i < fingertip_indices.size(); ++i)
+        F32 allowable_error = 0.016f;
+        ensure("LLIK::Solver finger should reach target", error < allowable_error);
+
+        // cleanup LLJoints
+        for (auto joint: joints)
         {
-            F32 error = dist_vec(solver.getJointWorldEndPos(fingertip_indices[i]), finger_target_positions[i]);
-            ensure("LLIK::Solver finger should reach target", error < allowable_error);
+            delete joint;
         }
     }
-#endif // ENABLE_FAILING_UNIT_TESTS
```

Comment:
I fixed this unit-test, however most of the unit tests are disabled right now in a big `#if 0` block while we investigate IK issues.